client-libraries/c/README.md

```diff
@@ -0,0 +1 @@
+# libmodality_client C API
```

Comment:
Would be nice to get a little bit of description in here. Maybe point to the `ctest` dir to see example use.

Reply:
Good call, will do

Reply:
Ended up deleting the README for now, will come back with a docs pass if needed